docs/src/other/changelog.md

```diff
@@ -7,6 +7,11 @@ description: |
 
 {{< description >}}
 
+## 2022
+
+* **January 2022**
```

Comment:
Technically we are in February, but I assume it refers to the date of implementation?

Reply:
That was my idea :shrug: